WeArtCommon.h

```diff
@@ -10,6 +10,9 @@
 #include <string>
 #include "nlohmann/json.hpp"
 
+#include <cstdint>   
+typedef uint8_t uint8;
```

Comment:
I would directly replace `uint8` with `uint8_t`, upstream seems to agree, see https://github.com/WEARTHaptics/WEART-SDK-Cpp/pull/1 . Perhaps we can try to rebase on last upstream master before opening a PR to upstream?

Reply:
Ah, actually we are already aligned. In that case, if there are still some `uint8` arond, perhaps we can just replace them with `uint8_t` ?

Reply:
Good point!

Reply:
In the end it was not necessary anymore. Removed it